test/cmdlineTests.sh

```diff
@@ -130,8 +131,12 @@ test_solc_behaviour() {
     exitCode=$?
     set -e
 
-    sed -i -e '/^Warning: This is a pre-release compiler version, please do not use it in production./d' "$stderr_path"
-    sed -i -e 's/ Consider adding "pragma .*$//' "$stderr_path"
+    if [[ "$standard_json" = "1" ]]; then
+        sed -i -e 's/,\{0,1\}{[^{]*Warning: This is a pre-release compiler version[^}]*},\{0,1\}//' "$stdout_path"
```

Comment:
If there is a `,` at the beginning and end, it will remove both, but I think it's fine.

Reply:
Or actually, I think we should remove the first `,`, this warning should always be the first.

Reply:
Ah, yes, true... but yes, I think it's fine for now and luckily we will notice this failing on the develop branch, not on the release branch.